### PR TITLE
added Jensen plot

### PR DESCRIPTION
**Pull request by AngRodrigues**
Added Jensen ternary plot for the classification of sub-alkaline volcanic rocks.

## Description
Followed the already existing .json template to generate the a new template for Jensen plot with the classification of sub-alkaline rocks.  

## Related Issue
No issue associated. 

## Motivation and Context
Jensen plot was not previously available in pyrolite, and I needed to use a Jensen plot in my code. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by morganjwilliams — CHANGES_REQUESTED** [review]
Hey @AngRodrigues, thanks for the PR, and nice work on your first! This runs locally and I've added a few minor things to address in comments here. 

It might be good to work up an example which can be added to the docs pages (and also used in a test), but we can do that subsequently as desired. I've added a specification for a quick synthetic dataset which isn't exactly 'natural' but otherwise demonstrates the space below (note that if axes names are changed, they should be changed here too..).

```python
from pyrolite.util.synthetic import normal_frame, random_cov_matrix

df = normal_frame(
        columns=["Ti + Fe(tot) cat %", "Al cat %", "Mg cat %"],
        mean=[0.07, 0.15, 0.12],
        cov=random_cov_matrix(2, sigmas=[0.5, 0.7]),
        size=100,
      )
```

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply .vscode/settings.json:1]: Sure, it has been removed from PR

File: pyrolite/data/models/JensenPlot/config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{"name": "JensenPlot", "axes": {"t": "Ti + Fe(tot) cat %", "l": "Al cat %", "r": "Mg cat %"}, "transform": "ternary",
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/data/models/JensenPlot/config.json:1]: changed to mol%

File: pyrolite/data/models/JensenPlot/config.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{"name": "JensenPlot", "axes": {"t": "Ti + Fe(tot) cat %", "l": "Al cat %", "r": "Mg cat %"}, "transform": "ternary", 
+    "fields": 
+    {"High Ti-Fe Other": {"name": "High Ti Fe Other", "poly": [[100, 0, 0], [85, 0, 15], [70, 15, 15], [85, 15, 0]]},
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/data/models/JensenPlot/config.json:3]: Yes, I was a bit confused on which labels are used in the figure; it's nice that both can be used. Updated as per suggested.

File: pyrolite/data/models/USDASoilTexture/config.json [hunk 4]
```diff
@@ -1 +1,17 @@
...], [0, 85, 15], [0, 70, 30], [15, 85, 0]]}, "sandy-loam": {"name": "Sandy Loam", "poly": [[0, 70, 30], [15, 85, 0], [20, 80, 0], [20, 52.5, 27.5], [7.5, 52.5, 40], [7.5, 42.5, 50], [0, 50, 50]]}, "sandy-clay-loam": {"name": "Sandy Clay Loam", "poly": [[20, 80, 0], [20, 52.5, 27.5], [27.5, 45, 27.5], [35, 45, 20], [35, 65, 0]]}, "sandy-clay": {"name": "Sandy Clay", "poly": [[35, 45, 20], [35, 65, 0], [55, 45, 0]]}, "loam": {"name": "Loam", "poly": [[20, 52.5, 27.5], [7.5, 52.5, 40], [7.5, 42.5, 50], [27.5, 22.5, 50], [27.5, 45, 27.5]]}, "clay-loam": {"name": "Clay Loam", "poly": [[27.5, 20, 52.5], [27.5, 45, 27.5], [40, 45, 15], [40, 20, 40]]}, "clay": {"name": "Clay", "poly": [[100, 0, 0], [55, 45, 0], [40, 45, 15], [40, 20, 40], [60, 0, 40]]}, "silty-clay": {"name": "Silty Clay", "poly": [[40, 20, 40], [60, 0, 40], [40, 0, 60]]}, "silty-clay-loam": {"name": "Silty Clay Loam", "poly": [[40, 20, 40], [40, 0, 60], [27.5, 0, 72.5], [27.5, 20, 52.5]]}, "silt-loam": {"name": "Silt Loam", "poly": [[27.5, 0, 72.5], [27.5, 22.5, 50], [0, 50, 50], [0, 20, 80], [12.5, 7.5, 80], [12.5, 0, 87.5]]}, "silt": {"name": "Silt", "poly": [[0, 20, 80], [12.5, 7.5, 80], [12.5, 0, 87.5], [0, 0, 100]]}}}
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/data/models/USDASoilTexture/config.json:1]: Agreed; reset to the previous version.

File: pyrolite/plot/templates/JensenPlot.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    color : :class:`str`
+        Line color for the diagram.
+
+    # Returns
+    # -------
+    # ax : :class:`matplotlib.axes.Axes`
+
+    # References
+    # -----------
+    # .. [#ref_1] Jensen, L. S. (1976) A new cation plot for classifying sub-alkaline volcanic rocks.
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/plot/templates/JensenPlot.py:33]: Done.

File: pyrolite/plot/templates/JensenPlot.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+    # Returns
+    # -------
+    # ax : :class:`matplotlib.axes.Axes`
+
+    # References
+    # -----------
+    # .. [#ref_1] Jensen, L. S. (1976) A new cation plot for classifying sub-alkaline volcanic rocks. 
+    #             Ontario Division Mines. Miscellaneous Paper No. 66.
+    # """
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/plot/templates/JensenPlot.py:35]: Nice catch! Done

File: pyrolite/util/classification.py [hunk 7]
```diff
@@ -572,6 +572,40 @@ def predict(self, df: pd.DataFrame):
...
+    -----------
+    name : :class:`str`
+        A name for the classifier model.
+    axes : :class:`list` | :class:`tuple`
+        Names of the axes corresponding to the polygon coordinates.
+    fields : :class:`dict`
+        Dictionary describing indiviudal polygons, with identifiers as keys and
+        dictionaries containing 'name' and 'fields' items.
+
+    References
```
[the remark this answers is not in the document]
Comment (AngRodrigues) [reply pyrolite/util/classification.py:590]: Added notes at the end. Not too familiar with more current approaches for this, but added some extra information.

**Pull request merged**